src/test/server/integration/content.test.ts

```diff
@@ -45,30 +45,29 @@ describe('(server/integration) Content fetch', () => {
             });
     });
 
-    it('restore content keys', (done) => {
-        const elGamalPublic = '7317752633383033582159088' +
-            '0415095934922384683502050702002361917832276924025919' +
-            '7334324222430627661202908079769675760465400935084759' +
-            '1901976526778157668840202.';
-        const privateKey = '5JDFQN3T8CFT1ynhgd5s574mTV9UPf9WamkHojBL4NgbhSBDmBj';
-        const keyPair = new dcore_js.KeyPair(privateKey, elGamalPublic);
-        const buyContentId = '2.12.114';
-        dcore_js.content().restoreContentKeys(buyContentId, accountId, keyPair)
-            .then(res => {
-                expect(res).a('string');
-                done();
-            })
-            .catch(err => {
-                expect(err).to.be.a('array');
-                done();
-            });
-    }).timeout(5000);
+    // it('restore content keys', (done) => {
```

Comment:
Same here - why comments?

Reply:
That is integration test for content restoration. However, due to stage testnet we do not have content, we cannot restore content keys.

Reply:
There are two options - explain this in at the beginning of comment section or remove test.